script/make-pretty

```diff
@@ -36,7 +36,8 @@
 #
 # Format c/c++ only:
 #
-#     script/make-pretty clang
+#     script/make-pretty clang-format
+#     script/make-pretty clang-tidy
```

Comment:
```suggestion
#     script/make-pretty clang
```

Can we enhance clang to include both clang-format and clang-tidy?
I think using `clang` to do both is more convenient to use.

Reply:
clang-tidy is quite a bit slower than clang-format. I think it might be useful to have clang-format separate.

Of course, we can have three options (clang, clang-format, clang-tidy), where clang is performs both clang-format and clang-tidy.

Thoughts?

Reply:
It's good to have three options.

Reply:
I agree, having 3 options with `clang` doing both sounds useful.